corehq/apps/reports/views.py

```diff
@@ -1477,9 +1477,9 @@ def clear_report_caches(request, domain):
 def export_report(request, domain, export_hash, format):
     cache = get_redis_client()
 
-    if cache.exists(export_hash):
```

Comment:
did you find that `exists` doesn't work? this should be in included in the new upgrade. tested it out locally as well